app/gui2/src/components/GraphEditor/__tests__/clipboard.test.ts

```diff
@@ -52,7 +54,7 @@ class MockClipboardItem {
     this.types = Object.keys(data)
   }
 
-  getType(type: string): Blob {
+  async getType(type: string): Promise<Blob> {
```

Comment:
`async` is not needed if there's no await AFAIK.

Reply:
`async` is the simplest way to turn the returned `Blob` into a `Promise`, which is necessary to match the mocked API